src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -376,3 +377,13 @@ def load_arguments(self, _):
         c.argument('name', arg_type=name_arg_type)
     with self.argument_context('functionapp config appsettings') as c:
         c.argument('slot_settings', nargs='+', help="space-separated slot app settings in a format of <name>=<value>")
+
+    with self.argument_context('functionapp devops-build') as c:
+        c.argument('functionapp_name', help="Name of the Azure Function App that you want to use", required=False)
+        c.argument('organization_name', help="Name of the Azure DevOps organization that you want to use", required=False)
+        c.argument('project_name', help="Name of the Azure DevOps project that you want to use", required=False)
+        c.argument('overwrite_yaml', help="If you have an existing yaml, should it be overwritten?", choices=['true', 'false'], required=False)
```

Comment:
Suggest built-in support `get_three_state_flag`

Reply:
Thanks for the suggestion. I have fixed and pushed